File: CS/CustomXRDesignRibbonForm.cs
using AIWizardCustomizationExample.Customization;
using DevExpress.XtraReports.UserDesigner;
using DevExpress.XtraReports.Wizards;

namespace AIWizardCustomizationExample {
    public class CustomXRDesignRibbonForm : XRDesignRibbonForm {
        public CustomXRDesignRibbonForm() {
            DesignMdiController.AddService(typeof(IWizardCustomizationService), new WizardCustomizationService());
        }

        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);
            DesignMdiController.CreateNewReportWizard();
        }
    }
}

File: VB/CustomXRDesignRibbonForm.vb
Imports AIWizardCustomizationExample.AIWizardCustomizationExample.Customization
Imports AIWizardCustomizationExample.Customization
Imports DevExpress.XtraReports.UserDesigner
Imports DevExpress.XtraReports.Wizards

Namespace AIWizardCustomizationExample

    Public Class CustomXRDesignRibbonForm
        Inherits XRDesignRibbonForm

        Public Sub New()
            DesignMdiController.AddService(GetType(DevExpress.XtraReports.Wizards.IWizardCustomizationService), New WizardCustomizationService())
        End Sub

        Protected Overrides Sub OnLoad(ByVal e As EventArgs)
            MyBase.OnLoad(e)
            DesignMdiController.CreateNewReportWizard()
        End Sub
    End Class
End Namespace
